src/bptl/conf/base.py

```diff
@@ -418,3 +419,9 @@
         "Token": {"type": "apiKey", "name": "Authorization", "in": "header"}
     },
 }
+
+# ValidSign Documents settings
+# The maximum size (in Mb) that all the documents combined can have
+MAX_TOT_DOCUMENT_SIZE = os.getenv("MAX_TOT_DOCUMENT_SIZE", 2000)
```

Comment:
2000 Mb is _a lot_ - that requires that your process is allowed to use 2GB of memory. I'd change that to about 250 Mb instead.